mythril/laser/ethereum/instructions.py

```diff
@@ -861,7 +845,7 @@ def jump_(self, global_state):
         except AttributeError:
             logging.debug("Invalid jump argument (symbolic address)")
             return []
-        except IndexError:  # Stack Underflow
+        except StackUnderflowException:
```

Comment:
I'm going to do my best to merge https://github.com/ConsenSys/mythril/pull/514/ in a minute or so. 
In which case we can remove this except :+1: